web/source/kmwosk.js

```diff
@@ -2501,88 +2899,72 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
       var layout;
 
       // Build a layout using the default for the device
-      var layoutType=formFactor,dfltLayout=keymanweb['dfltLayout'];
-      if(typeof dfltLayout[layoutType] != 'object') layoutType = 'desktop';
+      var layoutType=formFactor, dfltLayout=keymanweb['dfltLayout'];
+      if(typeof dfltLayout[layoutType] != 'object') {
+        layoutType = 'desktop';
+      }
 
       // Clone the default layout object for this device
       layout=util.deepCopy(dfltLayout[layoutType]);
 
-      var n,layers=layout['layer'],keyLabels=PVK['BK'],key102=PVK['K102'];
-      var i,j,k,m,row,rows,key,keys;
+      var kbdBitmask = PVK['KMBM'];
```

Comment:
I see this same code elsewhere. Can we refactor into a function `getKeyboardModifierBitmask()`?